packages/client/package.json

```diff
@@ -147,7 +147,7 @@
     }
   },
   "dependencies": {
-    "@prisma/engines-version": "4.4.0-47.c6b5fdac855d9b76c03ffc33c58188b56985fc6b"
+    "@prisma/engines-version": "4.4.0-55.qe-transaction-write-conflict-911f8bae8f9b08719066939ba0aaff82c1fe4a50"
```

Comment:
Note: switch to non-integration version before merge.

Reply:
Did that when solving conflicts (as I figured out that PR was merged)